### PR TITLE
[ft-benchmark] add note about db-sending scripts to README

### PR DESCRIPTION
**Pull request by MCJOHN974**

### CODE REVIEW

File: benchmarks/continous/db/README.md [hunk 1]
```diff
@@ -42,3 +42,21 @@ select setting from pg_settings where name = 'max_connections';
 ## Remote connection
 
 To connect to the database remotely, you can execute the `psql` recipe in the [`Justfile`](./Justfile).
+
+
+# Adding results of you benchmark automatically.
```
Comment (akashin) [inline benchmarks/continous/db/README.md:47]: Nit: Remove dot in the end of the heading for consistency

File: benchmarks/continous/db/README.md
Comment (akashin) [inline benchmarks/continous/db/README.md:1]: This is the documentation for the database for the benchmark, not the benchmark itself, so I think we need to move these instructions elsewhere. I see two options:
1. Add them to our outline document: https://docs.nearone.org/doc/crt-benchmark-8IpRzRV7C5/ - simpler short-term, but not as discoverable
2. Create a new document under Workflows section of developer documentation (https://near.github.io/nearcore/practices/workflows/index.html, https://github.com/near/nearcore/tree/master/docs/practices/workflows) - a proper place for the long-term

**Review by akashin — CHANGES_REQUESTED** [review]

File: benchmarks/continous/db/README.md
Comment (MCJOHN974) [reply benchmarks/continous/db/README.md:1]: What about just moving it to `benchmarks/continous/`? (And also move all ft-benchmark related scripts there)

**Comment by MCJOHN974** [comment]
https://github.com/near/nearcore/pull/11644 will replace it

**Pull request closed without merging**